dev-environment/create_environment.sh

```diff
@@ -1,12 +1,18 @@
 #!/bin/sh
 
+scalaV="2.12"
+nussknackerV="0.2.2"
+
 # Compile and package prinz if not packaged yet
 cd .. &&
-./sbtwrapper prinz/package prinz_sample/package &&
+./sbtwrapper prinz/package prinz_sample/package prinz_sample/pack &&
```

Comment:
do we need the package command when pack is used?

Reply:
I think that pack depends on package so it's unused here for prinz_simple

Reply:
Yes, it's for exporting dependency jars. Built-in `package` can't do anything like that.

Reply:
I mean that package command may be unused  here (as pack may depands on it and call it in background). Let's check it but commit is approved :wink:

Reply:
`pack` depends on `package` and I could remove call to `prinz_sample/package` eventually.